vector/src/main/res/values/strings.xml

```diff
@@ -2566,8 +2572,20 @@
     <string name="auth_pin_reset_title">Reset pin</string>
     <string name="auth_pin_new_pin_action">New pin</string>
     <string name="auth_pin_reset_content">To reset your PIN, you\'ll need to re-login and create a new one.</string>
+    <string name="settings_security_application_protection_title">Protect access to the application.</string>
+    <string name="settings_security_application_protection_summary">Protect access to the application using pin code and biometrics.</string>
```

Comment:
**Suggestion**
Protect access using PIN and biometrics.

Reply:
Done